src/main/java/net/sf/jabref/gui/preftabs/PreferencesDialog.java

```diff
@@ -42,6 +42,8 @@
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 
+import static net.sf.jabref.Globals.prefs;
```

Comment:
I'm not sure if we want to import this statically. I see the point, but it will be quite confusing compared to all other files where it is accessed as `Globals.prefs`...